moped-editor/src/views/projects/projectView/ProjectComponents/index.js

```diff
@@ -275,30 +286,24 @@ export default function MapView({
                 onSaveDraftComponent={onSaveDraftComponent}
                 onSaveEditedComponent={onSaveEditedComponent}
               />
+            </List>
+            {isNotCreatingOrEditing ? (
```

Comment:
Updating this to toggle rendering here instead of further down in the tree decreased the number of props here.